src/test/resources/kodeverk-spraak.json

```diff
@@ -0,0 +1,50 @@
+{
+  "betydninger": {
+    "HI": [
```

Comment:
Kanskje en case der det _ikke_ er en array? Siden det er implementert støtte for det

Reply:
yep yep, I agree...I didnt understood your comment immediately

Reply:
Done